woocommerce/woocommerce
# Allow multiple screenshots per test fix

### All Submissions:

* [ x ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [ x ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ x ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?





### Changes proposed in this Pull Request:

Updated `utils/take-screenshot.js` appending a string with the day and time `-${day}${hour}${minute}${second}` to the filename for screenshots taken.  Every time a failure occurs, a screenshot with a unique filename is taken.

Closes #31469 .

### How to test the changes in this Pull Request:

1. `$ export E2E_RETEST=1` (to enable retrying failed tests)
2. `pnpm nx docker-up woocommerce`
3. `pnpm nx test-e2e woocommerce` (I have a few tests that fail locally -- if everything is passing locally for you, you might have to force a test failure)
3. In `$ tests/e2e/screenshots` you should see screenshots for each retry of the test (up to 3)

### Other information:

* [ x ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ x ] Have you successfully run tests with your changes locally?



### FOR PR REVIEWER ONLY:
